caffe2/core/storage.h

```diff
@@ -32,6 +35,35 @@ class StorageImpl {
   explicit StorageImpl(DeviceType device_type) : device_type_(device_type) {}
   StorageImpl(DeviceType device_type, TypeMeta data_type)
       : data_type_(data_type), device_type_(device_type) {}
+  template <typename Deleter = MemoryDeleter>
+  StorageImpl(
+      DeviceType device_type,
+      TypeMeta data_type,
+      void* src,
+      TIndex size,
+      size_t capacity,
```

Comment:
Should document the difference between size and capacity prominently.

Reply:
updated the diff with size removed.